tests/test_eratos_sdk.py

```diff
@@ -2,7 +2,7 @@
 import os
 from eratos.creds import AccessTokenCreds
 import xarray as xr
-
+import numpy as np
```

Comment:
Optional NIT: I know you're only adding numpy but I think we can seperate the 1st and 3rd party import like so.

```diff
- import unittest
- import os
- from eratos.creds import AccessTokenCreds
- import xarray as xr
- 
- import numpy as np
+ import unittest
+ import os
+
+ import xarray as xr
+ import numpy as np
+ from eratos.creds import AccessTokenCreds
```